### PR TITLE
Learning strategies

### PR DESCRIPTION
**Pull request by tannerwelsh**
Inspired by some of the manifesto talk in Slack as well as a book I've been reading, I decided to capture that goodness and augment it with some of my own ideas about strategies.

Still first draft. Would like to include better citations and references to back up the theory. But better something than nothing, and it's been too long that we haven't had explicit learning strategies be a part of the environment.

### CODE REVIEW

File: Strategies/Team.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The less mutual clarity you have, the greater the chances are for misunderstanding and conflict, which is (usually) a cause of friction for everyone's learning.
+
+## Stay in Team
+
+Lead without slowing down. Follow without disengaging. Assert yourself. Stay flexible. Don't triangulate. Bring out the best in others.
+
+## Apply Cognitive Apprenticeship
+
+Every team has a variety of skills and skill levels. Use these internal differences to everyone's advantage: apply cognitive apprenticeship practices to best match team members to the tasks that will challenge them, and let the others (for whom the task would be too difficult) observe, reflect, and inquire.
+
```
Comment (shereefb) [inline Strategies/Team.md:30]: Link to cognitive apprenticeship

File: Strategies/Team.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Strategies for Team Learning
+
+A collection of our favorite strategies for learning in team.
+
+Roughly organized in order of preference: proven, high-leverage strategies are listed before less-important ones.
+
+If you have a strategy you'd like to add or edit, please consider [contributing](../CONTRIBUTING.md).
+
```
Comment (shereefb) [inline Strategies/Team.md:8]: Missing a section on pairing?

File: Strategies/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Learning strategies for [individuals](Individual.md), [teams](Team.md), and [communities](Community.md).
+
+These categories do not have strict boundaries between them: they are all complimentary and overlap in many places. We just use separate files to make it easier to find and reference them.
+
+## Why Strategies
+
+At Learners Guild, we are co-building an environment that is highly conducive to learning and growth.
+
+This is not easy.
+
```
Comment (shereefb) [inline Strategies/README.md:12]: We are used to authority being centralized. When authority is distributed, it means every participant has more power, and with it, more decisions and more responsibility. 

In the same way, a citizen in a democracy has to be better informed and make more decisions than someone in a kingdom: A member of a learning collective carries more weight and autonomy than a student in a traditional educational institution.

**Comment by shereefb** [comment]
Yes. Yes. Yes! Merge. Ship it. Do it. Thank you!

File: Strategies/Team.md (hunk 2 above)
Comment (tannerwelsh) [reply Strategies/Team.md:8]: Right! :) Will do.

**Pull request merged**